### PR TITLE
Add 22.10 backfill script[IN-6764]

### PR DESCRIPTION
**Pull request by hongjianniu**
Backfill JobOrganizationUnitId
https://uipath.atlassian.net/browse/IN-6764

### CODE REVIEW

File: Scripts/SQLServer/BackfillScripts/Backfill_22_10.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+select qie.[JobOrgUnitFullyQualifiedName] as qOrgName, qie.[JobOrganizationUnitId] as qOrgId, j.[OrganizationUnitId] as jOrgId, j.[OrgUnitFullyQualifiedName] as jOrgName
+from [dbo].[QueueItemEvents] qie 
+inner join [dbo].[QueueItems] qi on qie.[QueueItemId] = qi.[Id]
+inner join [dbo].[Jobs] j on qi.[ExecutorJobId] = j.[Id]
+where j.[OrganizationUnitId] != qie.[JobOrganizationUnitId] or (qie.[JobOrganizationUnitId] is null and j.[OrganizationUnitId] is not null))
+update CTE
+set qOrgName = jOrgName, qOrgId = jOrgId;
+
+--2.truncate read table
+truncate table [read].[QueueItemEvents];
```
Comment (max-shekhovtcov) [inline Scripts/SQLServer/BackfillScripts/Backfill_22_10.sql:12]: how do we make sure that there's no MigrateJobs/QueueItems already running when we run this script?

Comment (hongjianniu) [reply Scripts/SQLServer/BackfillScripts/Backfill_22_10.sql:12]: truncate wait the other statement finish before executing. I try to truncate a table while insert is running. Truncate doesn't start until insert finish.

**Review by GengyiQin — APPROVED** [review]

File: Scripts/SQLServer/BackfillScripts/Backfill_22_10.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+truncate table [read].[QueueItems];
+truncate table [read].[Jobs];
+truncate table [read].[JobEvents];
+delete [dbo].[IngestionMarkers] where [IngestionEventType] = 9006;
+delete [dbo].[IngestionMarkers] where [IngestionEventType] = 9001;
```
Comment (floringrigoriu) [inline Scripts/SQLServer/BackfillScripts/Backfill_22_10.sql]: nit how about combining in single query : delete [dbo].[IngestionMarkers] where [IngestionEventType] in ( 9006, 9001);

File: Scripts/SQLServer/BackfillScripts/Backfill_22_10.sql [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+where j.[OrganizationUnitId] != qie.[JobOrganizationUnitId] or (qie.[JobOrganizationUnitId] is null and j.[OrganizationUnitId] is not null))
+update CTE
+set qOrgName = jOrgName, qOrgId = jOrgId;
+
+--2.truncate read table
+truncate table [read].[QueueItemEvents];
+truncate table [read].[QueueItems];
+truncate table [read].[Jobs];
+truncate table [read].[JobEvents];
+delete [dbo].[IngestionMarkers] where [IngestionEventType] = 9006;
```
Comment (floringrigoriu) [inline Scripts/SQLServer/BackfillScripts/Backfill_22_10.sql]: in an slack thread @hongjianniu mentioned the value should be 9002 (EtlQueueItemsAndQueueItemEvents ) no 9006  (EtlDiscoveryQueueItemsJsonKey )

**Review by max-shekhovtcov — APPROVED** [review]

**Pull request merged**